src/LogEventProcessor.ts

```diff
@@ -19,6 +20,7 @@ const deduperInterval = 60 * 1000;
 export default class LogEventProcessor {
   private options: StatsigOptions;
   private fetcher: StatsigFetcher;
+  private loggingAdapter: ILoggingAdapter | null;
```

Comment:
rather than nullable, just make the default and in memory storage adapter, or a no op?  Should simplify things - that way there is always an adapter